tagstudio/src/qt/ts_qt.py

```diff
@@ -36,16 +36,17 @@
 							   QMenu, QTableWidget, QTableWidgetItem)
 from humanfriendly import format_timespan, format_size
 
-from src.core.library import Collation, Entry, ItemType, Library, Tag
-from src.core.palette import ColorType, get_tag_color
-from src.core.ts_core import (PLAINTEXT_TYPES, TagStudioCore, TAG_COLORS, DATE_FIELDS, TEXT_FIELDS, BOX_FIELDS, ALL_FILE_TYPES,
+from tagstudio.src.core.library import Collation, Entry, ItemType, Library, Tag
+from tagstudio.src.core.palette import ColorType, get_tag_color
+from tagstudio.src.core.ts_core import TagStudioCore
+from tagstudio.src.core.constants import (PLAINTEXT_TYPES, TAG_COLORS, DATE_FIELDS, TEXT_FIELDS, BOX_FIELDS, ALL_FILE_TYPES,
 										SHORTCUT_TYPES, PROGRAM_TYPES, ARCHIVE_TYPES, PRESENTATION_TYPES,
 										SPREADSHEET_TYPES, DOC_TYPES, AUDIO_TYPES, VIDEO_TYPES, IMAGE_TYPES,
 										LIBRARY_FILENAME, COLLAGE_FOLDER_NAME, BACKUP_FOLDER_NAME, TS_FOLDER_NAME,
 										VERSION_BRANCH, VERSION)
-from src.core.utils.web import strip_web_protocol
-from src.qt.flowlayout import FlowLayout, FlowWidget
-from src.qt.main_window import Ui_MainWindow
+from tagstudio.src.core.utils.web import strip_web_protocol
+from tagstudio.src.qt.flowlayout import FlowLayout, FlowWidget
+from tagstudio.src.qt.main_window import Ui_MainWindow
```

Comment:
`Tagstudio.src...` import format breaks at runtime for vscode and when running from the terminal

Reply:
Thank you for the feedback! I have made the changes.